src/Uno.UI/UI/Xaml/Media/AcrylicBrush.macOS.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using AppKit;
+using CoreAnimation;
+using CoreGraphics;
+
+namespace Windows.UI.Xaml.Media
+{
+	public partial class AcrylicBrush
+	{
+		internal void CreateAcrylicBrushLayers(
```

Comment:
This method should probably return a disposable so the added views get removed when the brush is changed